README.md

```diff
@@ -1,21 +1,40 @@
-![Integration Test](https://github.com/Azure/aml-deploy/workflows/Integration%20Test/badge.svg)
-![Lint](https://github.com/Azure/aml-deploy/workflows/Lint/badge.svg)
+![Integration Test](https://github.com/Azure/aml-deploy/workflows/Integration%20Test/badge.svg?branch=master&event=push)
+![Lint and Test](https://github.com/Azure/aml-deploy/workflows/Lint%20and%20Test/badge.svg?branch=master&event=push)
+
+# GitHub Action for deploying Machine Learning Models to Azure
 
-# Azure Machine Learning Deploy Action
 
 ## Usage
 
-The Azure Machine Learning Deploy action will deploy your model on Azure Machine Learning and create a real-time endpoint for use in other systems. The action currently supports Azure Container Instance and Azure Kubernetes Service as compute target and also supports the no-code deployment of Azure Machine Learning, if the model has been regiustered accordingly.
+The Deploy Machine Learning Models to Azure action will deploy your model on [Azure Machine Learning](https://azure.microsoft.com/en-us/services/machine-learning/) using GitHub Actions.
+
+Get started today with a [free Azure account](https://azure.com/free/open-source)!
+
+This repository contains GitHub Action for deploying Machine Learning Models to Azure Machine Learning and creates a real-time endpoint on the model to integrate models in other systems. The endpoint will be hosted either on an Azure Container Instance or on an Azure Kubernetes Service. 
+
+
+This GitHub Action also allows you to provide a python script that executes tests against the Webservice endpoint after the model deployment has completed successfully. You can enable tests by setting the parameter `test_enabled` to true. In addition to that, you have to provide a python script (default `code/test/test.py`) which includes a function (default ` def main(webservice):`) that describes your tests that you want to execute against the service object. The python script gets the [webservice object](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.webservice(class)?view=azure-ml-py) injected. The action fails, if the test script fails.
+
 
-This GitHub Action also allows you to provide a python script that executes tests against the  Webservice endpoint after the model deplyoment has completed successfully. You can enable tests by setting the parameter `test_enabled` to true. In addition to that, you have to provide a python script (default `code/test/test.py`) which includes a function (default ` def main(webservice):`) that describes your tests that you want to execute against the service object. The python script gets the [webservice object](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.webservice(class)?view=azure-ml-py) injected. The action fails, if the test script fails.
+## Dependencies on other GitHub Actions
+* [Checkout](https://github.com/actions/checkout) Checkout your Git repository content into GitHub Actions agent.

```

Comment:
still my point that there is difference between dependency and requirement. use can write his own checkout action.  so he is not dependent on this. 
Also not clear what is the dependency on aml-compute? and register model

Reply:
the details next to aml-compute here calls out it is conditionally needed. "You don't need this if you want to host your endpoint on an ACI instance" . Is there a recommendation you have on the title ?  Today it follows the pattern in other Az Actions. https://github.com/Azure/webapps-deploy#dependencies-on-other-github-actions

Reply:
I think, it will be fine if we can call out Using with other GitHub Actions, than dependency. 
just my opinion!

Reply:
@ashishonce maybe we can use something like: "Other actions required for building an AML MLOps solution"